lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -134,16 +143,22 @@ function dosomething_campaign_preprocess_common_vars(&$vars, &$wrapper) {
         ),
       ),
       'twitter' => array(
```

Comment:
@angaither one more question! I'm assuming we want to have the share bar here again with the icons but might look weird in the "stuff you need" section since the format of that is text in a bulleted list. Should we just add the share bar in this "STEP 2: PLAN IT" section, perhaps under the "Find a Location" section or just give the user their unique share URL (that might look very ugly just as text)?